### PR TITLE
[PRO-129] Add translation strings

### PR DESCRIPTION
**Pull request by ayreal**
## Description
This PR adds the translation strings for the project. There may be a couple things I missed but just poking around in the app I didn't notice any missing translation strings.

## Next Up
This PR sets up the structure for someone to take on the Spanish translations in [this ticket](https://linear.app/project-protocol/issue/PRO-203/provide-spanish-translations).

### CODE REVIEW

File: src/pages/Vote.tsx [hunk 1]
```diff
@@ -4,19 +4,23 @@ import { Document } from '@contentful/rich-text-types'
 import renderRichText from 'src/util/renderRichText'
 import icon from '../images/icon.svg'
 import BasicPage from 'src/components/BasicPage'
+import { useTranslation } from 'react-i18next'
 
 export default function Vote() {
+  const { t } = useTranslation()
+
+  // TODO: Handle translation
```
Comment (ayreal) [inline src/pages/Vote.tsx:12]: I didn't realize y'all were using a CMS. I wonder if [adding a locale in Contentful](https://www.contentful.com/help/working-with-translations/) will magically work here 😅 . My guess is it probably reads the browser's locale so we'd have to play around with configs to get it to read from local storage.

File: src/components/RateAgentModal/RateAgentTags.tsx [hunk 2]
```diff
@@ -18,6 +20,7 @@ export default function RateAgentTags({ control }: IRateAgentTags) {
       render={({ field }) => {
         const { value, onChange } = field
 
+        //TODO: Assure form value compatibility with translations
```
Comment (ayreal) [inline src/components/RateAgentModal/RateAgentTags.tsx:23]: I'm out of practice with react-hook-form so I'd want to see how the field values interact with the forms. The forms should always send through the English form values but display the translated copy. Right now I've left it as-is to handle in another PR.

Comment (Timmehs) [reply src/components/RateAgentModal/RateAgentTags.tsx:23]: Good catch. The translationMap sets this up well, we just need to make sure the value of that tag as used in handleClick is the key, and the displayed text is the translation. I'll open a PR after this one goes in.

**Review by Timmehs — APPROVED** [review]
Super thorough, looks great!

**Pull request merged**